docs/pages/platform_guides/version_control.mdx

```diff
@@ -26,45 +26,46 @@ By default, Dataform will manage your project's Git repository for you. However,
 
 One of the main benefits of Git is that a developer can work in an isolated "branch" of the file repository without affecting the base files. Dataform has full support for Git branches.
 
-From the `development` branch - or any other non-`production` branch - you can edit files (and run those edited scripts on your warehouse, if desired).
+From any non-`production` branch you can edit files (and run those edited scripts on your warehouse, if desired).
 
-Once you are happy with your changes, you can commit them and push them to the `production` branch. If at any point you make changes that you don't want to commit, you can revert your branch to its last commit.
+Once you are happy with your changes, you can commit them and push them to the `master` branch. If at any point you make changes that you don't want to commit, you can revert your branch to its last commit.
 
 Note that the `production` branch is not directly editable and **all schedules run from the production branch**. This enables users to develop and test changes without affecting the project's data pipelines or affecting other users.
 
-## How to use
+### Personal and shared branches
 
-By default, you will use the `development` branch. Anyone on your team can make edits here.
+When you first use Dataform we will automatically create a personal development branch for you, `firstname_dev`. Only you can see this branch and commit to it.
```

Comment:
firstname isn't quite right, it might be, it might not, depending on what data auth0 gives us. i would just say `name_dev`